src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
```

Comment:
Oh right. The problem with this implementation is that users of the Polyfill will be able to access and construct CSSImageValues using this constructor. You'll need to hide the internal constructor using some code construct like this: https://github.com/css-typed-om/typed-om/pull/135/files